### PR TITLE
Boot from flash instructions.

### PR DESCRIPTION
**Pull request by naveenm18**
Added documentation for boot from flash
using SPI/MEM.

Signed-off-by: Mutha <naga.naveen.mutha@intel.com>

### CODE REVIEW

File: source/developer-guides/flash-boot.rst [hunk 1]
```diff
@@ -0,0 +1,132 @@
...
+
+**Step 2:** Load Container image into PDR region.
+
+.. image:: /images/fit1.png
+   :width: 6.5in
+   :height: 3.17669in
+   :alt: Boot flash.
+   :align: center
+
+**Step 3:** Change Region order in Boot settings.
```
Comment (mauricema) [inline source/developer-guides/flash-boot.rst:111]: Why do we need to change the region order ?

Comment (naveenm18) [reply source/developer-guides/flash-boot.rst:111]: Existing region order is '45321' . 

After enabling PDR region with previous region order no logs on UART ,target not booted. When PDR is enabled and changed region order('54321') im able to boot it. 

1=BIOS, 2=ME/IFWI, 3=GbE, 4=PDR, 5=EC

Comment (mauricema) [reply source/developer-guides/flash-boot.rst:111]: I guess that is project specific thing.  If it does not work, might be some other issue.
Technically PDR region could be anywhere and it should work the same way.

Comment (naveenm18) [reply source/developer-guides/flash-boot.rst:111]: Ok i will revert this.

**Pull request merged**